sw/device/silicon_creator/rom/data/rom_e2e_testplan.hjson

```diff
@@ -18,6 +18,22 @@
       tests: ["rom_e2e_smoke"]
     }
 
+    {
+      name: rom_e2e_default_otp_bootup
+      desc: '''Verify that ROM can boot a ROM_EXT with default OTP values (all zeroes).
+
+            - Create an OTP with all zeros except:
+                - OWNER_SW_CFG_ROM_BOOTSTRAP_EN set to 0x739 (true)
+                - CREATOR_SW_CFG_ROM_EXEC_EN set to 0xffffffff (enabled)
+                - LC_STATE in the TEST_UNLOCKED0 state
+                - LIFE_CYCLE count value set to 1
+            - Ensure that the ROM can boot.
+            '''
+      tags: ["rom", "verilator", "dv", "fpga", "silicon"]
```

Comment:
was this supposed to be run in DV or Verilator? from the looks of the BUILD test target below, I would assume no? if not then this should read:
```
tags: ["rom", "no_dv", "fpga", "silicon"]
```

As is, it is showing up in the DV nightly dashboard as failing.

Reply:
Argh sorry about that -- I moved a bit too fast on this PR. I submitted #16704 to fix.

Reply:
@alphan Just to confirm, does this test need to be run in DV or Verilator at the moment?

Reply:
no problem, thanks for the fix!

Reply:
@milesdai no, it doesn't need to be run in DV or verilator.